ci/test-appimagetool.sh

```diff
@@ -117,3 +117,21 @@ if [ "$hash1" != "$hash2" ]; then
     echo "Hash $hash1 doesn't match hash $hash2!"
     exit 1
 fi
+
+log "check --mksquashfs-opt forwarding"
+out=$("$appimagetool" appimagetool.AppDir appimagetool.AppImage --mksquashfs-opt "-misspelt-option" 2>&1)
+echo "${out}" | grep -q "invalid option"
+"$appimagetool" appimagetool.AppDir appimagetool.AppImage.1
+"$appimagetool" appimagetool.AppDir appimagetool.AppImage.2 --mksquashfs-opt "-mem" --mksquashfs-opt "100M"
```

Comment:
A meaningful test should use maybe 1M or less, appimagetool is very small.

Reply:
This is a lot larger by default, here's the relevant output from `mksquashfs --help`:
```
-mem <size>		Use <size> physical memory.  Currently set to 3937M
			Optionally a suffix of K, M or G can be given to specify
			Kbytes, Mbytes or Gbytes respectively
```
I think this is just 25% of all available memory.
If you specify less than `16M`, mksquashfs complains: https://github.com/plougher/squashfs-tools/blob/188454662ed6de1332fe72dc119977b5ccb5fb1d/squashfs-tools/mksquashfs.c#L6617
So I figured 100M would be an ok value.

Reply:
That's a pity.

Currently we won't run into that limit. I think to actually test whether the option has any effect we should actually stress `mksquashfs` here.

On the other hand, we can never validate the option was passed anyway...